### DISCUSSION TITLE
Raspberry Pi support?

### DISCUSSION THREAD

**Discussion by seamusdemora** [Makuna/Rtc#147] [Q&A] [answered]
Apologies if this is off-topic, but this appears to be an extensive, well-maintained library for configuring the DS3231. I have dabbled in Arduino, but not since the **Uno** was a thing :)   I need a library like this for Raspberry Pi!  I'm posting here in an effort to learn if that might be forthcoming in this project, or if you know of a library that already exists. 

I need to be able to set/program alarms in the DS3231 - I need *state changes* on the `INT/SQW` output pin to interface to some hardware.

**Comment by Makuna** [comment, marked as answer]
(moved to discussions as it's not an issue)
This is an Arduino Library, if you have a Raspberry Pi that can be programmed using Arduino, then it should just work.  Like the RP2040.
There is no effort on this repo to make this compatible with non-Arduino targets.
